ProjectSource/src/main/java/com/kayako/sdk/base/requester/AbstractRequestProperty.java

```diff
@@ -0,0 +1,53 @@
+package com.kayako.sdk.base.requester;
+
+import java.util.Map;
+
+public abstract class AbstractRequestProperty extends GetRequestProperty {
+
+  private final String helpCenterUrl;
+  private final String endpointUrl;
+  private final Map<String, String> queryParameters;
+  private final Map<String, String> headers;
+  private final IncludeArgument include;
+  private final Map<String, String> bodyParameters;
+
+  AbstractRequestProperty(String endpointUrl, String helpCenterUrl,
+      Map<String, String> queryParameters, Map<String, String> headers, IncludeArgument include,
+      Map<String, String> bodyParameters) {   
+    this.endpointUrl = endpointUrl;
+    this.helpCenterUrl = helpCenterUrl;
+    this.queryParameters = queryParameters;
+    this.headers = headers;
+    this.include = include;
+    this.bodyParameters = bodyParameters;
+  }
+
+  public Map<String, String> getBodyParameters() {
+    return bodyParameters;
+  }
+
+  @Override
+  public String getHelpCenterUrl() {
```

Comment:
✅ It is nice to see the usage of @Override taking advantage of compiler checking to make sure that the code is actually overriding a method.

**No change suggested, looks good!**
 

<a id="c7d78d30-50c2-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />